pkgs/development/compilers/graalvm/community-edition.nix

```
@@ -1,48 +1,60 @@
-{ lib, stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath }:
+{ lib, stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath, darwin, openssl }:
```

```suggestion
{ lib, stdenv, fetchurl, perl, unzip, glibc, zlib, setJavaClassPath, Foundation, openssl }:
```

Please inherit Foundation from in top level. ``inherit (darwin.apple_sdk.frameworks) Foundation``

---

You forgot the inherit.

---

I don't understand your suggestion. How can I `inherit (darwin.apple_sdk.frameworks) Foundation` when `darwin` isn't in scope?

---

I mean something like this https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L1956.

---

I see, thanks, fixed!